docs/user-guide/admin-cli.md

```diff
@@ -408,7 +408,7 @@ optional arguments:
 Grants datastore access to a vmgroup.
 Valid value for "datastore" includes the name of valid datastores in the ESX host , special value "_VM_DS" or "_ALL_DS".
 When DS is set to _VM_DS, access to vm_datastore where vm lives is allowed for vms in vmgroup.
```

Comment:
Not the current change, but I think "vm_datastore" should be simply "datastore".